packages/commands/diff/package.json

```diff
@@ -33,6 +33,9 @@
     "@graphql-inspector/core": "0.0.0-PLACEHOLDER",
     "@graphql-inspector/commands": "0.0.0-PLACEHOLDER",
     "@graphql-inspector/logger": "0.0.0-PLACEHOLDER",
+    "express": "4.17.1",
+    "express-graphql": "0.11.0",
+    "graphql-tools": "6.2.4-alpha-34c1194a.0",
```

Comment:
@kamilkisiela This line could be removed when [this bug](https://github.com/ardatan/graphql-tools/issues/2051#issuecomment-699916227) is fixed.

@ardatan cc.

Reply:
@FluorescentHallucinogen looks good but let's wait for the fix @ardatan